demo/large_image_demo.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+from argparse import ArgumentParser
+
+import mmcv
+from mmdet.apis import inference_detector, init_detector
+from mmengine.logging import print_log
+from mmengine.utils import ProgressBar
+from sahi.slicing import slice_image
+
+from demo.large_image_demo_utils import merge_results_by_nms
+from mmyolo.registry import VISUALIZERS
+from mmyolo.utils import register_all_modules, switch_to_deploy
+from mmyolo.utils.misc import get_file_list
+
```

Comment:
You can write the specific usage through `docstr` in front

Reply:
or write the specific usage of this script in `docs/en/user_guides/useful_tools.md` please

Reply:
done